docs/source/releasenotes.rst

```diff
@@ -5,6 +5,7 @@ Release notes
 +++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
 
 - Library **RPA.Database**: Fix configuration value retrieval. (:pr:`456`)
+- Library **RPA.Dialogs**: Add next button to support wizard style dialogs. (:pr:`455`)
```

Comment:
Here you can put your issue like so:
```suggestion
- Library **RPA.Dialogs**: Add next button to support wizard style dialogs. (:issue:`452`)
```

Forgot to mention that issues take precedence in front of PRs. (if an issue exists ofc, for simpler hotfixes sometimes we do PRs directly, but never pushing directly to master :D)